localgovdrupal/localgov_services
# Remove the override releated links from service pages

Fix #72



## What does this change?

This is a holdover from the old BHCC site, but the functionality for automated links was never implemented in Localgov Drupal.

This will remove this field for new sites, and will always render the related links if they are present regardless.



## How to test

Do a fresh install, create a service page. 
There should no longer be a replace automatically generated related links.
When adding a related link, this will now show on a service page all the time.
Before, if the above was not checked, then related links would not display, since the auto generation function was never implemented in LGD.



## How can we measure success?

Related links now work as an editor would expect.



## Have we considered potential risks?

Some councils may be using this 'feature' to hide related links and not show them, we should check that.



## Images




## Accessibility
n/a
